client/public/js/actions/EventActions.js

```diff
@@ -22,8 +21,16 @@ var EventActions = {
     });
   },
 
+  getAllEventsBySheep: function(sheepId) {
```

Comment:
Got it. Do you mind putting a comment to note that so that we know to disregard it for the time being?